README.md

```diff
@@ -35,34 +9,33 @@ https://developer.github.com/v3/oauth/#web-application-flow
 
 *Or*, if you just need to get up and running *fast*, follow these simple steps:
 
-### 1. Install `hapi-auth-github` from NPM
+### 1. Install `hapi-github-auth` from NPM
 
 Install the plugin from npm and save it to your `package.json`:
 
 ```sh
-npm install hapi-auth-github --save
+npm install hapi-github-auth --save
 ```
 
 ### 2. Create an App on GitHub
 
 Follow the instructions in:
-[GITHUB-APP-STEP-BY-STEP-GUIDE.md](https://github.com/dwyl/hapi-auth-github/blob/master/GITHUB-APP-STEP-BY-STEP-GUIDE.md)
+[GITHUB-APP-STEP-BY-STEP-GUIDE.md](https://github.com/maixiaojie/hapi-github-auth/blob/master/GITHUB-APP-STEP-BY-STEP-GUIDE.md)
```

Comment:
Given this and the fact that you've deleted a lot of the readme/code, I'm going to assume you didn't mean to push this as a PR to this repo.